config/pipelines/high_throughput_heron.yml

```diff
@@ -2,7 +2,7 @@
 Heron:
   filters:
     request_type_key: limber_heron
-    library_type: Heron
+    library_type: PCR amplicon ligated adapters
```

Comment:
Assume capitals don't matter? :)

Reply:
Yeah David put 'PCR Amplicon ligated adapters' but I thought it was inconsistent having a capital 'A' and not the other ones so I made them all lower case

Reply:
Just a string I think, that is what they asked for.